src/jni-module.cpp

```diff
@@ -81,17 +81,19 @@ DLLEXPORT qore_license_t qore_module_license = QL_MIT;
 DLLEXPORT char qore_module_license_str[] = "MIT";
 
 // module cmd type
-typedef void (qore_jni_module_cmd_t)(QoreString& arg);
-static void qore_jni_mc_import(QoreString& arg);
-static void qore_jni_mc_add_classpath(QoreString& arg);
-static void qore_jni_mc_add_relative_classpath(QoreString& arg);
+typedef void (qore_jni_module_cmd_t)(const QoreString& arg);
```

Comment:
fyi you can use `using` instead of typedefs, for functions it's easily readable and it supports templates https://en.cppreference.com/w/cpp/language/type_alias
```
typedef void (*func)(int, int);
using func = void (*) (int, int);
```

Reply:
ok done - did not know about that - much better than a `typedef`!